README.md

```diff
@@ -219,6 +219,21 @@ A configuration that can be passed to the [bell][bell] OAuth module to authentic
 #### Expected Promise Response
 1. Resolve with a valid [bell][bell] configuration
 
+### getWebhookStrategy
+
+#### Expected Outcome
+A hapi webhook auth strategy object that can be passed to the hapi server.
```

Comment:
Wouldn't it be easier to just implement the ~10 lines of validation required https://github.com/mhazy/hapi-github-webhooks/blob/master/lib/hmac.js#L11-L17 by passing in the entire headers to the parseHook function.